ConvivaExampleApp/src/main/java/com/bitmovin/analytics/convivaanalyticsexample/MainActivity.java

```diff
@@ -8,6 +8,7 @@
 import android.widget.Switch;
```

Comment:
@irishsheila Should this PR be going into develop rather than master?

Reply:
Good catch. Fixed.